roles/nfs_volumes/tasks/nfs.yml

```diff
@@ -0,0 +1,27 @@
+---
+- name: Install NFS server on Fedora/Red Hat
+  yum: name=nfs-utils state=installed
+  when: ansible_os_family == "RedHat"
+
+- name: Install NFS server on Debian/Ubuntu
+  apt: name=nfs-kernel-server state=installed
+  when: ansible_os_family == "Debian"
```

Comment:
Do we need the os related tests here?  The module metadata only lists Fedora and EL 7 as supported.